frontend/views/containers/payments/ExportPaymentsModal.vue

```diff
@@ -0,0 +1,188 @@
+<template lang='pug'>
+modal-template(ref='modal' :a11yTitle='modalTitle')
+  template(slot='title')
+    span {{ modalTitle }}
+
+  i18n.c-sub-title.has-text-1(:args='{ type: paymentType }') Export your {type} payment history to .csv
```

Comment:
Minor request, for translators who just see this string by itself without comment or context, it would help to know what `{type}` might mean.

So could you rename this variable to `{sent_or_received}` ?

Reply:
Actually, I'm realizing even this is wrong.

There are multiple problems with doing it this way:

1. `paymentType` isn't localized, so this is completely broken.
2. Even if `paymentType` were localized, it's still better to avoid inserted directly translated words or fragments into another string, but instead it is better to preserve the entire string and translate it as a whole. It's not always the case that if you correctly translate "sent" that that isolated translation will fit when inserted into the sentence "export your [sent] payment..." the direct translation of the word "sent" might need to be something different in that context.

So here I recommend using a computed property instead of `i18n`, and use `paymentType` to pick the full sentence:

```js
exportInstructions () {
  return this.paymentType === 'sent'
    ? L('Export your sent payment history to .csv')
    : L('Export your received payment history to .csv')
}
```

This is much safer and guarantees a correct translation.

Reply:
@taoeffect yeah, great catch on this one. it could've lead to an awkwardly translated sentence.